prow/cluster/monitoring/mixins/prometheus/tide_alerts.libsonnet

```diff
@@ -0,0 +1,55 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'Tide progress',
+        rules: [
+          {
+            alert: 'TideNoProgressKK',
+            expr: |||
+              clamp_min(
```

Comment:
I don't think they are mutually exclusive. We should probably do both for best results?
I also don't want to publish 0s unless we can distinguish them with their own histogram bucket: https://github.com/kubernetes/test-infra/blob/64c95ffab14b71c4d1a71f95ff4aba3b4dfd9140/prow/tide/tide.go#L169
Can we retrospectively add buckets to the beginning of the range? (I know we can add them to the end, but I'm not sure if we can add them anywhere else because the buckets are cumulative.)

Reply:
Not sure we can, actually. Good point